test/protocol/ExchangeHandlerTest.js

```diff
@@ -5424,6 +5586,79 @@ describe("IBosonExchangeHandler", function () {
             // It should match ExchangeState.Disputed
             assert.equal(response, ExchangeState.Disputed, "Exchange state is incorrect");
           });
+
+          it("Too many twins", async function () {
```

Comment:
```suggestion
          it("Too many twins - erc20 & erc721 twins", async function () {
```

Reply:
I don't think this is necessary - this test is inside the context `Offer bundled with mixed twins` where all tests include a combination of different twins.
And it also uses erc115 twins.